### PR TITLE
#400 check call get account

### PR DESCRIPTION
**Pull request by phongvu2812**
# Ticket
https://github.com/mezonai/mmn-tx-explorer/issues/400

# General
- [x] No unnecessary files, debug logs, or commented code included.
- [ ] PR has at least 1 peer approval before senior review.
- [x] I fully understand the code I submitted.
- [x] No hardcoded configuration or secrets.
- [ ] **Environment Variables** changes? (Leave unchecked if N/A)

# Design & Solution Clarity
- [ ] A design diagram (sequence / flow) is attached or linked.
- [ ] The implementation matches the design.
- [ ] All major flows and decision points are represented in the diagram.
- [ ] Error handling and edge cases are clearly defined in the design.

Design link (diagram / doc):
`<paste link here>`

# Testing
- [ ] Unit tests cover main logic and important edge cases.

> Note: Complex logic must not rely on manual testing only.

# UI evidence (For frontend only)
image
iPhone-14-Plus-localhost

# Peer Review Checklist
- [ ] I understand the overall solution and flow.
- [ ] The code follows the provided design diagram.
- [ ] No obvious logic or security issues found.
- [ ] Tests are sufficient for the scope of this change.

### CODE REVIEW

File: frontend/modules/p2p/components/p2p-trading/p2p-offers-list.tsx [hunk 1]
```diff
@@ -74,7 +76,7 @@ export const P2POffersTabs = ({ offers, isLoading = false }: P2POffersTableProps
         <Button onClick={() => {
           router.push(ROUTES.P2P_TRADING_ROOM(offer.offer_id, 'offer'))
         }} className="rounded-lg bg-emerald-500 px-6 py-2 font-bold text-white transition hover:bg-emerald-600">
-          Buy đồng
+          Buy Mezon đồng
```
Comment (potatoArthurDo) [inline frontend/modules/p2p/components/p2p-trading/p2p-offers-list.tsx]: you have not hidden the button for offer owner

Image

File: frontend/modules/p2p/components/p2p-trading/mobile/offer-card.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ const OfferMobileCard = ({ offer }: OfferMobileCardProps) => {
 
       <div className="pt-1">
         <Button className="w-full rounded-xl bg-emerald-500 py-3 font-bold text-white shadow-md transition hover:bg-emerald-600 active:scale-[0.98]">
-          Buy {APP_CONFIG.CHAIN_SYMBOL}
+          Buy Mezon Đồng
```
Comment (potatoArthurDo) [inline frontend/modules/p2p/components/p2p-trading/mobile/offer-card.tsx]: hide the button for offer owner on mobile

**Comment by potatoArthurDo** [comment]
not from the files in this PR but let's resolve this first or nothing will run 
image

File: frontend/modules/p2p/components/p2p-trading/p2p-offers-list.tsx (hunk 1 above)
Comment (phongvu2812) [reply frontend/modules/p2p/components/p2p-trading/p2p-offers-list.tsx]: done

File: frontend/modules/p2p/components/p2p-trading/mobile/offer-card.tsx (hunk 2 above)
Comment (phongvu2812) [reply frontend/modules/p2p/components/p2p-trading/mobile/offer-card.tsx]: done

**Review by hoangdohuy — APPROVED** [review]
LGTM
